includes/class.llms.user.permissions.php

```diff
@@ -159,6 +159,57 @@ public static function get_editable_roles() {
 
 	}
 
+	/**
+	 * Modify a users ability to `view_grades`
+	 *
+	 * Users can view the grades (quiz results) if one of the following conditions is met:
+	 *   + Users can view their own grades.
+	 *   + Admins and LMS Managers can view anyone's grade.
+	 *   + Any user who has been explicitly granted the `view_grades` cap can view anyone's grade (via custom code).
+	 *   + Any user who can `edit_post` for the course the quiz belongs to can view grades of the students within that course.
```

Comment:
according to the code... this is true assuming the user is an instructor or an instructor assistant, right?

Reply:
Yes, updating the comment thanks